components/index.js

```diff
@@ -1,5 +1,5 @@
-import MoviesThumb from './movieThumb';
-import MoviesList from './moviesList';
+import MoviesThumb from './movieThumb/movieThumb';
+import MoviesList from './moviesList/moviesList';
 import Spinner from './spinner';
-import MovieDetails from './movieDetails';
-import SearchMovies from './searchMovies';
\ No newline at end of file
+import MovieDetails from './movieDetails/movieDetails';
+import SearchMovies from './searchMovies/searchMovies';
```

Comment:
you don't need this file it such form.
If you want to import components from components folder - define exports.
```
export * from './movieDetails';
```